app/utils/ReceiptDataHelper.scala

```diff
@@ -85,12 +85,11 @@ class ReceiptDataHelper @Inject()(implicit val messages: MessagesApi) extends Lo
           )
         )
         case None =>
-          logger.warn("[ReceiptDataHelper][extractDeclaration] Client name missing")
```

Comment:
same here

Reply:
this can be treated as validation error since the name model is there but empty